src/main/kotlin/apply/application/AdministratorService.kt

```diff
@@ -1,14 +1,38 @@
 package apply.application
 
+import apply.domain.administrator.Administrator
 import apply.domain.administrator.AdministratorRepository
+import apply.domain.administrator.getById
+import apply.domain.administrator.getByUsername
 import org.springframework.security.core.userdetails.UserDetails
 import org.springframework.security.core.userdetails.UserDetailsService
-import org.springframework.security.core.userdetails.UsernameNotFoundException
 import org.springframework.stereotype.Service
 
 @Service
```

Comment:
r: `@Transactional`이 필요하지 않을까요?

```suggestion
@Transactional
@Service
```